server/graphql/programs/src/queries/program_enrolment.rs

```diff
@@ -65,14 +65,15 @@ pub struct ProgramEnrolmentFilterInput {
 impl ProgramEnrolmentFilterInput {
     pub fn to_domain_filter(self) -> ProgramEnrolmentFilter {
         ProgramEnrolmentFilter {
-            context: self.program.map(EqualFilter::from),
             patient_id: self.patient_id.map(EqualFilter::from),
             enrolment_datetime: self.enrolment_datetime.map(DatetimeFilter::from),
             program_enrolment_id: self.program_enrolment_id.map(EqualFilter::from),
             status: self
                 .status
                 .map(|s| map_filter!(s, ProgramEnrolmentNodeStatus::to_domain)),
             document_name: self.document_name.map(EqualFilter::from),
+            document_type: self.program.map(EqualFilter::from),
+            context: None,
```

Comment:
Will this change from None in 1944?

Reply:
It will stay hidden from the API, i.e. context is only used internally... It's just used connected independent tables together, e.g. permissions, documents, reports, etc...